draft-ietf-ohai-ohttp.md

```diff
@@ -129,17 +130,22 @@ to correlate behavior. This imposes considerable performance and efficiency over
 to the additional round trip to the server (at a minumum), additional data exchanged, and
 additional CPU cost of cryptographic computations.
 
-This document defines two kinds of HTTP resources -- Oblivious Relay Resources
-and Oblivious Gateway Resources -- that process encapsulated binary HTTP messages
-{{BINARY}} using Hybrid Public Key Encryption (HPKE; {{!HPKE=RFC9180}}). They can be composed to
-protect the content of encapsulated requests and responses, thereby separating the identity of a
-requester from the request.
+To overcome these limitations, this document defines how binary
+HTTP messages {{BINARY}} can be encapsulated using Hybrid Public Key Encryption
+(HPKE; {{HPKE}}) to protect their contents. Clients exchange these messages with
+an Oblivious Gateway Resource, which is responsible for forwarding decapsulated
+requests to the original Target Resource and encrypting the corresponding
+responses and sending them back to the client. Critically, the encapsulated
```

Comment:
```suggestion
responses with encryption and sending them back to the client. Critically, the encapsulated
```

Reply:
That would have it say "encrypting the corresponding responses with encryption".  Did you mean to suggest "encapsulating the corresponding responses [with encryption] and sending them back to the client" ?  I don't think you need the "with encryption" here.

Reply:
Yeah, sorry, I meant it to say "encapsulating the request with encryption" (or whatever).